### ISSUE TITLE
Add Accessibility to the front-end tree

### ISSUE THREAD

**Issue by wBatdeveloppement4** [kamranahmedse/developer-roadmap#432]
Hi Kamran, great work, thank you.

I think the front-end part lacks accessibility / WAI / WCAG / ARIA…, perhaps even on the legal (i.e. anti-discrimination laws) side.

**Comment by kamranahmedse** [comment]
This has been fixed and will be published in the coming release. Thank you 🙏
